src/Core/Domain/Configuration/Command/EnableDebugModeCommand.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * 2007-2019 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Domain\Configuration\Command;
+
+/**
+ * Enables debug mode if it's not yet enabled.
+ */
+class EnableDebugModeCommand
```

Comment:
I'm considering the idea of turning this `EnableDebugModeCommand` into `SwicthDebugModeCommand` which would be able, using a boolean input, to enable or disable the debug mode. Wdyt @sarjon ?

Reply:
hmmm, i guess it could be. what values it should accept?

Reply:
Just a boolean, set to "true" to enable and to "false" to disable. It does the job, like https://github.com/PrestaShop/PrestaShop/pull/11894/files#diff-b3ae8c4352f26f962b5fb157163d6f6e